src/main/scala/com/evenfinancial/sbt/secrets/KeybaseSecrets.scala

```diff
@@ -1,6 +1,7 @@
 package com.evenfinancial.sbt.secrets
 
 import sbt._
+import scala.sys.process._
```

Comment:
What's this for? Do you need the whole package or can you import a specific class?